addons/l10n_sa_pos/static/src/overrides/models/pos_order.js

```diff
@@ -70,9 +70,6 @@ https://zatca.gov.sa/ar/E-Invoicing/SystemsDevelopers/Documents/20210528_ZATCA_E
         return name_tag_encoding.concat(name_length_encoding, name_byte_array);
     },
     get isSimplified() {
-        return (
-            (this?.partner_id?.company_type === "person" || !this?.partner_id) &&
-            this.company_id.country_id?.code === "SA"
-        );
+        return !this?.partner_id && this.company_id.country_id?.code === "SA";
```

Comment:
```suggestion
        return !this?.partner_id?.vat && this.company_id.country_id?.code === "SA";
```

If only a real company can have vat it should be like that.